composer.json

```diff
@@ -25,7 +25,7 @@
         "doctrine/cache": "^1.0",
         "doctrine/collections": "^1.0",
         "doctrine/event-manager": "^1.0",
-        "doctrine/reflection": "^1.1"
+        "doctrine/reflection": "^1.2"
```

Comment:
`^1.1` already allows installing 1.2.0 and higher. Is your goal to make sure 1.1 can no longer be installed in new versions of persistence?

Reply:
@greg0ire no my goal is that users automatically get the latest version. We had a few people report the same bugs again in ORM even though the fix is in doctrine/reflection, which they didn't upgrade to the latest version.

Reply:
Ok, not sure this will help a lot, but it can't hurt 👍